Makefile

```diff
@@ -150,6 +150,7 @@ mocks: $(MOCKS_DIR)
 	$(mockgen_cmd) -package mocks -destination tests/mocks/grpc_server.go github.com/gogo/protobuf/grpc Server
 	$(mockgen_cmd) -package mocks -destination tests/mocks/tendermint_tendermint_libs_log_DB.go github.com/tendermint/tendermint/libs/log Logger
 	$(mockgen_cmd) -source=orm/model/ormtable/hooks.go -package ormmocks -destination orm/testing/ormmocks/hooks.go
+	$(mockgen_cmd) -source=x/nft/expected_keepers.go -package testutil -destination x/nft/testutil/expected_keepers_mocks.go
```

Comment:
Assuming we go with this approach, the `mocks` make target is gonna get pretty out of hand -- I recommend we move this to a script.

Reply:
Go generate is considered best practice afaik